vividus-plugin-rest-api/src/test/java/org/vividus/bdd/steps/api/HttpRequestStepsTests.java

```diff
@@ -149,6 +157,35 @@ void testSetCustomRequestConfigNoField()
         assertEquals("nonExistentField", exception.getMessage());
     }
 
+    @Test
+    void testWaitForResponseCode()
+    {
+        SubSteps stepsToExecute = mock(SubSteps.class);
+        when(httpTestContext.getResponse()).thenReturn(new HttpResponse());
+        httpRequestSteps.waitForResponseCode(RESPONSE_CODE, Duration.ofSeconds(10), RETRY_TIMES, stepsToExecute);
```

Comment:
decrease duration and number of retries to some minimal value (10 seconds - too much for unit test)